app/services/submission_payload_service.rb

```diff
@@ -15,7 +15,7 @@ def user_answers_map
     questions = {}
     submission.fetch('sections', []).each do |section|
       section.fetch('questions').each do |question|
-        questions[question.fetch('key')] = question.fetch('answer')
+        questions[question.fetch('key')] = question.fetch('answer', '')
```

Comment:
I missed the conversation - what does this change do? Default to "empty string"?